apollo-ios/Sources/Apollo/ApolloClient.swift

```diff
@@ -49,22 +51,40 @@ public class ApolloClient {
   ///
   /// - Parameters:
   ///   - networkTransport: A network transport used to send operations to a server.
-  ///   - store: A store used as a local cache. Note that if the `NetworkTransport` or any of its dependencies takes a store, you should make sure the same store is passed here so that it can be cleared properly.
-  public init(networkTransport: any NetworkTransport, store: ApolloStore) {
+  ///   - store: A store used as a local cache. Note that if the `NetworkTransport` or any of its dependencies takes
+  ///   a store, you should make sure the same store is passed here so that it can be cleared properly.
+  ///   - sendClientMetadataExtension: Specifies whether client library metadata is sent in each request `extensions`
+  ///   key. Client library metadata is the Apollo iOS library name and version. Defaults to `true`.
```

Comment:
Sorry I'm late to the party. I'm guessing we went with `sendClientMetadataExtension` instead of just `enhanced client awareness` because we are keeping the door open to adding more things in those metadatas?

If that is the case, can we have a link to a public specification of what the `metadata` contains? This would help users understand what exactly is being sent and what tradeoff they are doing by disabling the feature.

Reply:
I'm not opposed to renaming it but was there a discussion somewhere about the naming?

Reply:
I don't think so.